tracr/io/dicom/read.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+"""
+A script that converts dicom files to an intensity voxel array
+The resulting array is 1000 x 1024 in cross section direction and 1014 "tall"
+"""
+
+import sys, os, re
+import numpy as np
+import dicom
+from errno import ENOENT
+
+def _regularize_dcm_files(dcm_files):
+	# Are we reading a file, a list of files, or files from a directory?
+	# 1. Is dcm_files a string?
+	if isinstance(dcm_files, str):
+		# yes: either a single file or a directory
+		if os.path.isdir(dcm_files):
+			# create a list of DICOM files from the contents of the directory
+			dcm_files = dcm_files.rstrip('/')
+			rval = ['{}/{}'.format(dcm_files, fname)
+					for fname in os.listdir(dcm_files)
+					if re.match(r'.*\.dcm', fname)]
```

Comment:
Just a note that [`glob`](https://docs.python.org/2/library/glob.html) can do the listing and filtering for you.